ci/ci/github.py

```diff
@@ -776,6 +777,10 @@ async def _start_deploy(self, batch_client, steps=()):
 (cd {shq(repo_dir)}; {self.checkout_script()})
 '''
             )
+            if sha_must_be:
+                out, err = sync_check_shell_output(f'(cd {repo_dir}; git rev-parse {self.sha})')
```

Comment:
What's the difference between `sync_check_shell_output` and `await chek_shell`?

Reply:
Don't think there is a difference besides that it returns stdout/stderr. It's also not async, but that's irrelevant for us here.